client/okteto.yml

```diff
@@ -4,11 +4,16 @@ labels:
   faas_function: bitcoin-studio-client
 command:
   - sh
-workdir: /home/app
-mountpath: /home/app/function
-#persistentVolume:
-#  enabled: false
+workdir: /home/app/function
+persistentVolume:
+  enabled: true
 securityContext:
   runAsUser: 998
   runAsGroup: 998
-  fsGroup: 998
\ No newline at end of file
+  fsGroup: 998
+resources:
```

Comment:
a little bit more memory so the build takes less time, you'll need to reduce this value if you want to run more than one dev env at the same time.